Topshelf.Linux/Topshelf.Linux.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Topshelf.Linux</RootNamespace>
     <AssemblyName>Topshelf.Linux</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```

Comment:
Please try to keep framework dependency at 4.0 (or the reasonable minimum supported by TopShelf)